src/resources/Catalogs/ProductListingInterfaces.ts

```diff
@@ -39,7 +39,7 @@ export interface ProductListingModel {
     /**
      * The list of exclusion filters.
      */
-    exlusion?: QueryFilterModel[];
+    exclude?: QueryFilterModel[];
```

Comment:
I think the commit should be flagged as a breaking changes? 🤔

Reply:
I don't think the interface is used anywhere yet, but it can be done to be safe.

Reply:
If it's a typo, it probably mean it never worked before. In that case, it's a fix.